examples/openssh.rs

```diff
@@ -0,0 +1,23 @@
+//! Expected `Cargo.toml` dependencies configuration:
+//!
+//! ```
+//! [dependencies]
+//! openssh-sftp-client = { version = "0.13.6", features = ["openssh"] }
```

Comment:
Maybe we should also add `tokio` and `openssh`.

Reply:
Thanks, I've added it.